pages/success-banner/index.js

```diff
@@ -175,17 +217,22 @@ const Footer = () =>
  * <Page />
  */
 const Page = () =>
-  <div>
-    <Head>
-      <meta charset="utf-8" />
-      <link
-        rel="stylesheet"
-        type="text/css"
-        href="/static/success-banner.css"
-      />
-    </Head>
-    <Banner />
-    <Footer />
-  </div>;
+  <Tracker analytics={locals.analytics}>
+    <div>
+      <Head>
+        <script dangerouslySetInnerHTML={{ __html: HEAD_SCRIPT }} />
```

Comment:
@Shou Do you know if this will work inside the "restricted" webview that Etcher uses?

Reply:
Good question @Shou do you think it will work? How would I test it locally?

Reply:
Run `etcher-homepage` locally and open it in a browser. How it displays there is how it'll look in Etcher as it's completely isolated – if you want to see it in Etcher itself then you can change [the URL on line 70 in `lib/gui/index.html`](https://github.com/resin-io/etcher/blob/master/lib/gui/index.html#L70) to the localhost one and that should work fine.